sdk/storagesync/Readme.md

```diff
@@ -0,0 +1,147 @@
+
+# Developer Handbook for StorageSync team
```

Comment:
Remove this file.   The README.md file for your library must be under `Azure.ResourceManager.StorageSync` and follow repository norms.